docs/articles/documentation/using-testcafe/common-concepts/browsers/user-profiles.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+title: User Profiles
+permalink: /documentation/using-testcafe/common-concepts/browsers/user-profiles.html
+---
+# User Profiles
+
+By default, TestCafe launches browsers with a clean profile, which means empty local storage, no extensions or bookmarks, etc. This was done to minimize the influence of profile intrinsics on test runs.
```

Comment:
TestCafe always runs tests with an empty local/session storage, so it doesn't depend on user profiles.